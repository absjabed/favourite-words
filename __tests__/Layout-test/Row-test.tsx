import 'react-native';
import React from 'react';
import Row from '../../src/screens/Layout/Row';
import renderer, { create } from 'react-test-renderer';

it('Row component renders correctly', () => {
  renderer.create(<Row />);
});

test('Row snapshot', () => {
  const tree = create(<Row/>);
  expect(tree).toMatchSnapshot();
});
