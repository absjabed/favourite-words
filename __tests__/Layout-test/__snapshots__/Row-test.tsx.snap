// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "width": "100%",
      },
      undefined,
    ]
  }
/>
`;
